assets/js/components/DeregistrationModal/DeregistrationModal.stories.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+
+import Button from '@components/Button';
+import DeregistrationModal from '.';
+
+export default {
+  title: 'DeregistrationModal',
+  component: DeregistrationModal,
+
+  args: {
+    hostName: 'example host',
+    isOpen: false,
+  },
+
+  argTypes: {
+    hostName: {
+      type: 'string',
+      description: 'The host name to confirm deregistration of',
+      control: { type: 'text' },
+    },
+    isOpen: {
+      type: 'boolean',
+      description: 'Sets the visibility of the modal',
+      control: false,
+    },
+    onCleanUp: {
+      description: 'Callback function to run when "Clean up" button is clicked',
+      action: 'Deregistration',
+      control: false,
+    },
+    onClose: {
+      description: 'Callback function to run when "Cancel" button is clicked',
+      action: 'Cancel',
+      control: false,
+    },
+  },
+};
+
+export function Default({ hostName, onCleanUp, onClose }) {
```

Comment:
Let's use the new format for declaring stories:
https://github.com/trento-project/web/blob/main/assets/js/components/DottedPagination/DottedPagination.stories.jsx#L28

https://storybook.js.org/docs/react/writing-stories/stories-for-multiple-components

Otherwise we cannot manage the args